Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindowUtils.cs

```diff
@@ -0,0 +1,39 @@
+#if UNITY_EDITOR && UNITY_INPUT_SYSTEM_UI_TK_ASSET_EDITOR
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem.HighLevel.Editor;
+using UnityEngine.UIElements;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    public class InputActionsEditorWindowUtils
+    {
+        public static StyleSheet theme => EditorGUIUtility.isProSkin
+        ? AssetDatabase.LoadAssetAtPath<StyleSheet>(InputActionsEditorConstants.PackagePath + InputActionsEditorConstants.ResourcesPath + "/InputAssetEditorDark.uss")
+        : AssetDatabase.LoadAssetAtPath<StyleSheet>(InputActionsEditorConstants.PackagePath + InputActionsEditorConstants.ResourcesPath + "/InputAssetEditorLight.uss");
+
+        public static void SaveAsset(SerializedObject serializedAsset)
+        {
+            var asset = (InputActionAsset)serializedAsset.targetObject;
+            if (asset.name == HighLevel.Input.kGlobalActionsAssetName)
```

Comment:
Maybe add some comments here to remind your future self what is going on here (that it's handling 2 cases)